app/caais/test/unit/test_models.py

```diff
@@ -697,4 +781,129 @@ def test_flatten_metadata_no_related_atom_2_1(self):
         self.assertEqual(flat["title"], "Title")
         self.assertEqual(flat["acquisitionType"], "")
 
-        metadata.delete()
+    @patch("django.conf.settings.CAAIS_UNKNOWN_DATE_TEXT", "Unknown")
+    @patch("django.conf.settings.CAAIS_UNKNOWN_START_DATE", "1900-01-01")
+    @patch("django.conf.settings.CAAIS_UNKNOWN_END_DATE", "1900-12-31")
+    def test_parse_event_date_for_atom_empty_date(self) -> None:
+        """Test parsing an empty date."""
+        metadata = Metadata(date_of_materials="")
+        metadata.save()
+
+        date_text, start_date, end_date = metadata.parse_event_date_for_atom()
+
+        self.assertEqual(date_text, "Unknown")
+        self.assertEqual(start_date, datetime.strptime("1900-01-01", "%Y-%m-%d").date())
+        self.assertEqual(end_date, datetime.strptime("1900-12-31", "%Y-%m-%d").date())
```

Comment:
You can construct dates using the `date` constructor:

```suggestion
        self.assertEqual(start_date, date(1900, 1, 1))
        self.assertEqual(end_date, date(1900, 12, 31))
```

You'll need `from datetime import date` at the top of the file.

This applies to this test and the rest of the tests in this class

Reply:
Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/448/commits/c499615b5617f307f3e273820679eafcd4dbfe62